packages/blade/src/components/List/listTokens.ts

```diff
@@ -0,0 +1,142 @@
+import type { ListProps } from './List';
+import type { TypographyPlatforms } from '~tokens/global/typography';
+import type { DotNotationSpacingStringToken } from '~src/_helpers/types';
+
+const listItemPaddingLeft: Record<
+  NonNullable<ListProps['variant'] | 'unorderedWithIcon'>,
+  Record<number, DotNotationSpacingStringToken>
+> = {
+  unordered: {
+    1: 'spacing.2',
+    2: 'spacing.5',
+    3: 'spacing.5',
+  },
+  unorderedWithIcon: {
+    1: 'spacing.0',
+    2: 'spacing.5',
+    3: 'spacing.5',
+  },
+  ordered: {
+    1: 'spacing.0',
+    2: 'spacing.6',
+    3: 'spacing.6',
+  },
+  'ordered-filled': {
+    1: 'spacing.0',
+    2: 'spacing.6',
+    3: 'spacing.6',
+  },
+};
+
+const listItemBulletPaddingRight: Record<
+  NonNullable<ListProps['variant']>,
+  DotNotationSpacingStringToken
+> = {
+  unordered: 'spacing.3',
+  ordered: 'spacing.2',
+  'ordered-filled': 'spacing.3',
+};
+
+const listItemBulletMarginTop: Record<
+  NonNullable<ListProps['variant'] | 'unorderedWithIcon'>,
+  Record<
+    TypographyPlatforms,
+    Record<NonNullable<ListProps['size']>, number | DotNotationSpacingStringToken>
+  >
+> = {
+  unordered: {
+    onDesktop: {
+      small: 6,
+      medium: 'spacing.3',
+    },
+    onMobile: {
+      small: 6,
+      medium: 10,
```

Comment:
For these tokens in some places we are using `number` but in some places `{number}px`, let's use only 1 format for hardcoded tokens?

Reply:
So wherever the spacing value matched our tokens, I opted to use our tokens. Its only the cases where we can't use our tokens we'll hard code those values.

Reply:
wherever we are not using our tokens why not plainly use hardcoded px value directly? for eg: `small: '6px',`?

Reply:
Because our `Box` component does that for us. Box accepts a `number` or a `DotNotationSpacingValue`. If we pass a number, it'll run `makeSpace` on it